src/blas/impl/KokkosBlas2_team_gemv_spec.hpp

```diff
@@ -92,6 +91,31 @@ struct TeamGEMV<TeamType, MatrixType, XVector, YVector, 0, false> {
   }
 };
 
+template <class TeamType, class MatrixType, class XVector, class YVector>
+struct TeamGEMV<TeamType, MatrixType, XVector, YVector, -1, false> {
```

Comment:
Thanks for adding the complex specialization

Reply:
No problem. I used (-1) so that the specialization stays close to 0 (which is the normal case).